### PR TITLE
Add hybrid pro to device selector python window

### PR DESCRIPTION
**Pull request by Vsevolod1983**
### PURPOSE
1. Add "hybrid pro" option to device selector python window.
2. Add optional "isModal" flag for dialog.

### EFFECT OF CHANGE
When you open device configuration window "HybridPro" option appears

### CODE REVIEW

**Review by bnagirniak — APPROVED** [review]

File: pxr/imaging/rprUsd/devicesConfiguration.py [hunk 1]
```diff
@@ -361,7 +363,7 @@ def on_device_change(self):
 
 
 class _DevicesConfigurationDialog(QtWidgets.QDialog):
-    def __init__(self, configuration, show_restart_warning):
+    def __init__(self, configuration, show_restart_warning, isModal):
```
Comment (s1lentssh) [inline pxr/imaging/rprUsd/devicesConfiguration.py:366]: Please use PEP8 naming convention, e.g. is_modal

File: pxr/imaging/rprUsd/devicesConfiguration.py [hunk 2]
```diff
@@ -398,6 +400,8 @@ def __init__(self, configuration, show_restart_warning):
         self.button_box.accepted.connect(self.on_accept)
         self.button_box.rejected.connect(self.on_reject)
         self.main_layout.addWidget(self.button_box)
+        if isModal:
```
Comment (s1lentssh) [inline pxr/imaging/rprUsd/devicesConfiguration.py:403]: Please use PEP8 naming convention, e.g. is_modal

File: pxr/imaging/rprUsd/devicesConfiguration.py [hunk 3]
```diff
@@ -422,7 +426,7 @@ def on_plugin_configuration_change(self):
                 self.restart_warning_label.hide()
 
 
-def open_window(parent=None, parent_flags=QtCore.Qt.Widget, show_restart_warning=True):
+def open_window(parent=None, parent_flags=QtCore.Qt.Widget, show_restart_warning=True, isModal=False):
```
Comment (s1lentssh) [inline pxr/imaging/rprUsd/devicesConfiguration.py:429]: Please use PEP8 naming convention, e.g. is_modal

File: pxr/imaging/rprUsd/devicesConfiguration.py [hunk 4]
```diff
@@ -441,7 +445,7 @@ def show_error(title, text):
     configuration_filepath = RprUsd.Config.GetDeviceConfigurationFilepath()
     configuration = _Configuration.load(Context(parent, parent_flags), configuration_filepath)
 
-    dialog = _DevicesConfigurationDialog(configuration, show_restart_warning)
+    dialog = _DevicesConfigurationDialog(configuration, show_restart_warning, isModal)
```
Comment (s1lentssh) [inline pxr/imaging/rprUsd/devicesConfiguration.py:448]: Please use PEP8 naming convention, e.g. is_modal

**Review by s1lentssh — CHANGES_REQUESTED** [review]

**Review by s1lentssh — APPROVED** [review]

**Pull request merged**